docs/swagger_v3/names.spec.yaml

```diff
@@ -102,13 +103,8 @@ schemas:
         type: integer
         example: 362026
       pointers:
-        type: array
-        items:
-          $ref: '#/components/schemas/Pointer'
-        properties: # TODO: can't be array and object at the same time
-          account_pubkey:
```

Comment:
I propose to remove this because autorest replaces it with `accountPubkey` which is very confusing.